### ISSUE TITLE
No 'version' key in the manifest file for custom integration 'display'.

### ISSUE THREAD

**Issue by ccorderod** [daemondazz/homeassistant-displays#23]
As of Home Assistant 2021.4.0, all components should have a version attribute on the manifest file.

This shows up on the log:

No 'version' key in the manifest file for custom integration 'display'. As of Home Assistant 2021.6, this integration will no longer be loaded. Please report this to the maintainer of 'display'

Thank you.

**Comment by piotrdal** [comment]
In the custom components folder for the integration you will have a file called manifest.json
You can add a version key to it yourself.

```
{
    "domain": "fully_kiosk",
    "name": "Fully Kiosk Browser",
    "documentation": "https://github.com/daemondazz/homeassistant-displays",
    "dependencies": [],
    "codeowners": ["@daemondazz"],
    "requirements": ["requests"],
    "version": "0.0.1"
}
```

Take particular note that all lines except the last one must have a comma at the end.
Restart HA and it should work ;)

**Comment by daemondazz** [comment]
My apologies folks, I added this to my local copy back in March but somehow overlooked committing and pushing. Fixed in commit 80d79cc.
